Dynamic validation example (#111) (#173)
GazzolaLab/PyElastica: examples/README.md

* [BoundaryConditionsCases](./BoundaryConditionsCases)
    * __Purpose__: Demonstrate the usage of boundary conditions for constraining the movement of the system.
    * __Features__: GeneralConstraint, CosseratRod
* [DynamicCantileverCase](./DynamicCantileverCase)

>     * __Purpose__: Physical validation of dynamic cantilever vibration at multiple modes.

Reviewer: ```suggestion
    * __Purpose__: Validation of dynamic cantilever vibration for multiple modes.
```
